script_automation/esa_ubuntu_k3_single/_index.md

```diff
@@ -0,0 +1,84 @@
+# Edge Storage Accelerator Single Node K3s on Ubuntu
+This example can be used to install ESA on an Ubuntu system with K3s. 
+
+> ⚠️ **Disclaimer:** The Edge Storage Accelerator is currently in public preview and not generally available. Access to the feature may be limited and subject to specific terms and conditions. For further details and updates on availability, please refer to the [Edge Storage Accelerator Documentation](https://learn.microsoft.com/azure/azure-arc/edge-storage-accelerator/overview).
+
+## Getting Started
+![Edge Storage Accelerator Diagram.](esa_diagram.jpg)
+
+## Prerequisites
+* Ubuntu 22.04 or similar VM or hardware that meets [ESA requirements](https://learn.microsoft.com/en-us/azure/azure-arc/edge-storage-accelerator/prepare-linux#minimum-hardware-requirements)
+  * Standard_D8ds_v4 VM recommended
+  * Equivalent specifications per node:
+    * 4 CPUs
+    * 16GB RAM
+  * 14G of free disk space in /var
+
+* Installation of [K3s](https://docs.k3s.io/quick-start)
```

Comment:
Which verison of k3s?

Reply:
We've never required a specific version; we've tried several older and much older versions and never had an issue.